packages/components/pie-toast/src/toast.scss

```diff
@@ -1 +1,71 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
```

Comment:
It isn't a realistic problem but worth mentioning that when multiline, if the message is a string with no spaces, it won't break to a new line ever